spec/subjects/async-subject-spec.js

```diff
@@ -1,19 +1,23 @@
 var Rx = require('../../dist/cjs/Rx');
 
 var AsyncSubject = Rx.AsyncSubject;
-var nextTick = Rx.Scheduler.nextTick;
 var Observable = Rx.Observable;
 
 describe('AsyncSubject', function () {
+  var results = [];
```

Comment:
I agree with you guys. I was also not very happy reusing var. I'll fix it and update this PR.